src/mutation.cpp

```diff
@@ -458,61 +458,37 @@ void player::activate_mutation( const trait_id &mut )
         }
         // Oops, no room to divide!
         if (valid.size() == 0) {
-            add_msg_if_player(m_bad, _("You focus, but are too hemmed in to birth a new slimespring!"));
+            if ( mut == trait_SLIMESPAWNER ) {
+                add_msg_if_player(m_bad, _("You focus, but are too hemmed in to birth a new slimespring!"));
+            } else {
+                add_msg_if_player(m_bad, _("You focus, but are too hemmed in to birth a new slimeling!"));
+            }
```

Comment:
Why this duplicate messages? If I'm not mistaken, player can't have `SLIMESPAWNER` and `SLIMESPAWNER2` at the same time? If so, remove the if-else check and show the message for either of the mutations.

Reply:
They can have both at the same time. `SLIMESPAWNER` is a requirement for `SLIMESPAWNER2`, but `SLIMESPAWNER` does not turn into `SLIMESPAWNER2`.